### ISSUE TITLE
How the data is processed and generated with which data and methods.

### ISSUE THREAD

**Issue by 123duduke** [HKUDS/RLMRec#7]
**trn_mat.pkl、tst_mat.pkl、val_mat.pkl**
Hello, how did you get these three sparse matrices? What data and methods did you use? It seems like you don't have this part of the code? Could you briefly introduce it? And how did you divide the obtained sparse matrices into training, testing, and validation parts? I would like to do this preprocessing work? Thank you very much.

**Comment by 123duduke** [comment]
This question, I seem to have understood, the sparse matrix is based on the user, item interaction data, but I have a new question, that is, why do you build the json file with a lot of formats\ n and\

**Comment by Re-bin** [comment]
Hi 👋

Thank you for your interest! The JSON file is written with one Python Dictionary per line using `json.dumps()`. To convert these lines into dictionary objects in Python, you can use the `json.loads()` function to parse each line. Thank you for your understanding :)

Best regards,
Xubin

**Comment by IIM-D** [comment]
Hello！ 👋
First question: Are the training set, test set, and validation set merely adjacency matrices of interactions? In other words, when replacing the dataset, is it sufficient to only create adjacency matrices composed of 0/1? 

Second question: I noticed that the number of users and items in the original dataset is relatively similar. If the number of users is significantly smaller than the number of items, would this affect the model from the perspectives of collaborative filtering and mutual information maximization?
